lib/react_client/react_interop.dart

```diff
@@ -507,13 +507,7 @@ class ReactDartInteropStatics2 implements ReactDartInteropStatics {
       JsMap jsPrevState,
     )
         handleGetDerivedStateFromProps,
-    bool Function(
-      Component2 component,
-      JsMap jsNextProps,
-      JsMap jsNextState,
-      dynamic jsNextContext,
-    )
-        handleShouldComponentUpdate,
+    bool Function(Component2 component, JsMap jsNextProps, JsMap jsNextState) handleShouldComponentUpdate,
     // TODO: Should this be removed when we update Component2.componentWillUpdate to throw an UnsupportedError? (CPLAT-4766)
```

Comment:
Is this TODO still valid?

Reply:
It was on the wrong line, but there was some leftover code from otehr lifecycle methods that I cleaned up in 06bce3f2e3b140d362a40e18f18729e0e1a4c8c8